src/Spatial.Tests/Euclidean/PlaneTest.cs

```diff
@@ -80,6 +85,77 @@ public void ProjectPointOn(string ps, string rootPoint, string unitVector, strin
             AssertGeometry.AreEqual(expected, projectedPoint, eps);
         }
 
+        [TestCase(ZeroPoint, X, Y)] //all 3 points are on the plane z=0. normal vector is +ez.
+        [TestCase(ZeroPoint, Y, Z)] //all 3 points are on the plane x=0. normal vector is +ex.
+        [TestCase(ZeroPoint, Z, X)] //all 3 points are on the plane y=0. normal vector is +ey.
+        public void CreateFittedPlaneFrom_PointsShouldBeOnTheFittedPlane(string sP1, string sP2, string sP3)
```

Comment:
I would also add another test case, with 3 general points, not just the ones which lie on the x, y and z axes.